regression_tests/tests/workflow_comparer.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import os
+import subprocess
+import sys
+from io import StringIO
+from contextlib import redirect_stdout
+
+from aqueduct import Client, get_apikey
+
+"""
+This script is used in regression testing to compare workflows.
+
+The checkpoint flag specifies the action to take
+create - stores workflow info as a checkpoint
+diff - compares the current state of the workflow with the last stored checkpoint
+
+Current comparisons are comparing the description of the flow each time for equality
```

Comment:
super nit: can we add periods and colons to the end of comment sentences? It's a tiny thing but I think it definitely helps with readability.